src/Compilers/Extension/Roslyn.Compilers.Extension.csproj

```diff
@@ -123,26 +123,8 @@
   </ItemGroup>
   <ItemGroup>
     <!-- Manual exclusions for obsolete interop assemblies -->
-    <!--<PackageReference Include="EnvDTE" Version="$(EnvDTEVersion)" ExcludeAssets="all" />-->
```

Comment:
Remove those properties from versions.props

Reply:
I'm removing all the ones I can, but this one cannot.

Reply:
OK now they are removed!